playground/nextjs/middleware.ts

```diff
@@ -8,6 +8,8 @@ const isPublic = (path: string) => {
   return publicPaths.find(x => path.match(new RegExp(`^${x}$`.replace('*$', '($|/)'))));
 };
 export default withClerkMiddleware(req => {
+  const { debug } = getAuth(req);
+  console.log('middleware:debug', debug());
```

Comment:
❓Should we keep the console logs? 
This is inside the playground app, and it makes sense to have it. I just want to confirm that the logs are not leftovers by mistake

Reply:
I think we should keep them. i added them intentionally.

Reply:
I figured, good call :)